### PR TITLE
HBW-75 Fix limits on the number of lines from sql

### PR DESCRIPTION
**Pull request by mxsmirnoff**

### CODE REVIEW

File: hbw/app/models/hbw/sources/oracle.rb [hunk 1]
```diff
@@ -4,18 +4,17 @@ class Oracle < Base
...
-      MAX_ROW_COUNT = 20
 
       def initialize(*)
         require 'oci8' unless defined?(OCI8)
 
         super
       end
 
-      def select(sql, variables)
+      def select(sql, variables, limit)
```
Comment (sugarfree1) [inline hbw/app/models/hbw/sources/oracle.rb]: There is a way to provide default argument values like so:
```
def select(sql, variables, limit = nil)
```

File: hbw/app/models/hbw/sources/oracle.rb [hunk 2]
```diff
@@ -4,18 +4,17 @@ class Oracle < Base
...
         binds = parse_bind_names(sql)
 
-        execute(limit(sql)) do |cursor|
+        inner_select(sql, limit) do |cursor|
```
Comment (sugarfree1) [inline hbw/app/models/hbw/sources/oracle.rb]: Rename ```inner_select ``` to ```execute_with_limit```.

**Comment by mxsmirnoff** [comment]
Updated

File: CHANGELOG.md [hunk 3]
```diff
@@ -14,12 +14,14 @@ v1.5.0 [unreleased]
 
 ### Bugfixes
 -   [#170](https://github.com/latera/homs/pull/170) Use active support logger as base.
+-   [#176](https://github.com/latera/homs/pull/176) Fix limits on the number of lines from sql requests.
```
Comment (sugarfree1) [inline CHANGELOG.md]: We don't duplicate lines. It is better to leave earlier version.

**Comment by mxsmirnoff** [comment]
Updated

**Pull request merged**